<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetronomeInterface</class>
 <widget class="QWidget" name="MetronomeInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>节拍器BPM</string>
  </property>
  <widget class="CardWidget" name="MetronomeCard">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>841</width>
     <height>460</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <widget class="ElevatedCardWidget" name="ElevatedCardWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>621</width>
      <height>145</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>621</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="LargeTitleLabel" name="bpmLargeTitleLabel">
        <property name="text">
         <string>___._</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="BodyLabel" name="bpmLabel">
        <property name="text">
         <string>BPM</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="SubtitleLabel" name="hintLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>841</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>按空格键测速</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="TitleLabel" name="TitleLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>49</y>
      <width>841</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>节拍器BPM</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="PushButton" name="reButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>400</y>
      <width>102</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ElevatedCardWidget</class>
   <extends>SimpleCardWidget</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SimpleCardWidget</class>
   <extends>CardWidget</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SubtitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LargeTitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
